build.py

```diff
@@ -660,7 +640,6 @@ def main():
         build_theme(ctx)
         logger.info("Done!")
```

Comment:
Ideally this log would be moved out and only printed once we're actually done. Maybe replace it with an empty log so we can more easily separate each build in the logs?